pipelines/hf_radar/config/dataset_6km_ref.yaml

```diff
@@ -77,15 +77,15 @@ coords:
     dtype: datetime64[s]
     attrs:
       units: Seconds since 1970-01-01 00:00:00
-  lat:
+  latitude:
```

Comment:
Can you confirm he wants latitude/longitude instead of lat/lon? Just went through a whole thing with wfip3 correcting those to the shorthand

Reply:
Last time I met with Raghu, he verified to update only this pipeline (from lat/lon to latitude/longitude) to make it consistent with other Oracle pipelines.